circuit_puzzles/programs/surplus_start_auction.clsp

```diff
@@ -1,22 +1,28 @@
 (mod
   (
-    (MOD_HASH CAT_MOD_HASH CAT_TAIL_HASH CRT_TAIL_HASH LAST_BID LAUNCHER_ID CAT_LOT_AMOUNT STATUTES_STRUCT)
+    (MOD_HASH CAT_MOD_HASH BYC_TAIL_HASH CRT_TAIL_HASH LAST_BID LAUNCHER_ID BYC_LOT_AMOUNT STATUTES_STRUCT)
```

Comment:
why I use CAT_TAIL_HASH is to also indicate that people can use whatever tail here, they're not forced to use BYC